supabase/supabase
# Chore/fix limit in sql query

fetch first was conflicting with our auto-appended limit


**To test:** 
this should fail before and pass after `select * from countries fetch first 5 rows only` 

 ![screenshot-2024-06-11-at-10 19 20](https://github.com/supabase/supabase/assets/105593/9e203ddb-25e9-4327-b201-f5c610d5b4a0)
